src/app/auth/features/user-manager-detail/user-manager-detail.component.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * User Manager Detail component styles
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+@use '@angular/material' as mat;
+
+:host {
+  display: block;
+  padding-bottom: 1em;
+  background-color: var(--mat-sys-surface-container-low);
+}
+
```

Comment:
Why not simply have a div wrapping the if and else and give it these classes? It's the same effect and avoids having to use `:host`.

Reply:
Removed this, since the detail page in this page is only meant as a paceholder and starting point anyway.